client/question/question.js

```diff
@@ -20,6 +20,101 @@ window.tie = angular.module('tie',
   ['ui.codemirror', 'tieConfig', 'tieData', 'ngCookies']);
 
 /**
+ * String used to label test suites that contain sample input and output
+ * that are presented to the user.
+ *
+ * @type {string}
+ */
+tie.constant('TEST_SUITE_ID_SAMPLE_INPUT', 'SAMPLE_INPUT');
+
+/**
+ * Correctness state for first user attempt to pass a test suite.
+ *
+ * @type {string}
+ */
+tie.constant('CORRECTNESS_STATE_STARTING', 'STARTING');
+
+/**
+ * Correctness state where the test case input has been displayed to the user.
+ *
+ * @type {string}
+ */
+tie.constant('CORRECTNESS_STATE_INPUT_DISPLAYED', 'INPUT_DISPLAYED');
+
+/**
+ * Correctness state where the expected output has been displayed to the user.
+ *
+ * @type {string}
+ */
+tie.constant('CORRECTNESS_STATE_EXPECTED_OUTPUT_DISPLAYED',
+  'EXPECTED_OUTPUT_DISPLAYED');
+
+/**
+ * Correctness state where the observed output is available to the user.
+ *
+ * @type {string}
+ */
+tie.constant('CORRECTNESS_STATE_OBSERVED_OUTPUT_AVAILABLE',
+  'OBSERVED_OUTPUT_DISPLAYED');
+
+/**
+ * Label for feedback displayed to user when presenting input(s) the user should
+ * consider before they revise their code.
+ *
+ * @type {string}
+ */
+tie.constant('FEEDBACK_TYPE_INPUT_TO_TRY', 'INPUT_TO_TRY');
+
+/**
+ * Label for feedback displayed to user when presenting the expected output for
+ * given input.
+ *
+ * @type {string}
+ */
+tie.constant('FEEDBACK_TYPE_EXPECTED_OUTPUT', 'EXPECTED_OUTPUT');
+
+/**
+ * Label for feedback displayed to user when allowing the user to display the
+ * output from their submitted code.
+ *
+ * @type {string}
+ */
+tie.constant('FEEDBACK_TYPE_OUTPUT_ENABLED', 'OUTPUT_ENABLED');
+
+/**
+ * Object containing variations on feedback text for different scenarios.
+ *
+ * @type {Object.<string, Array.<string>>}
+ */
+tie.constant('CORRECTNESS_FEEDBACK_TEXT', {
+  INPUT_TO_TRY: [
+    'Would your code work for the following input?',
+    'How about the following input? Would your code still work?',
+    'Would your code work with the following?',
```

Comment:
What happens when you run your code with this input? rather than this line.

Reply:
Revised as suggested.